GirafAPI/Endpoints/OrganizationEndpoints.cs

```diff
@@ -25,14 +25,14 @@ public static RouteGroupBuilder MapOrganizationEndpoints(this WebApplication app
                         return Results.BadRequest("Invalid user id.");
                     }
 
-                    await dbContext.Entry(user)
-                        .Collection(u => u.Organizations).LoadAsync();
-
                     if (user.Organizations is null)
```

Comment:
Har du testet det her? user.Organizations bliver først hentet når du kalder LoadAsync(), og så er jeg bange for at det her endpoint altid returnerer NotFound selvom organisationen findes.

Reply:
Altså det er nøjagtig samme kode, ! gør bare at den ignorer warning som ASP NET sender hver gang man laver en dotnet run